plugins/metadata/lib/kMetadataObjectCopiedHandler.php

```diff
@@ -183,9 +183,40 @@ public function objectCreated(BaseObject $object)
 	 */
 	public function objectChanged(BaseObject $object, array $modifiedColumns)
 	{
-		/* @var $object Permission */
-		$partner = PartnerPeer::retrieveByPK($object->getPartnerId());
-		$this->partnerPermissionEnabled($partner);
+		if($object instanceof Permission)
+		{
+			/* @var $object Permission */
+			$partner = PartnerPeer::retrieveByPK($object->getPartnerId());
+			$this->partnerPermissionEnabled($partner);
+		}
+
+		if($object instanceof entry)
+		{
+			$tempEntryId = $object->getId();
+			$partnerId = $object->getPartnerId();
+	
+			$metadataObjects = MetadataPeer::retrieveAllByObject(MetadataObjectType::ENTRY , $object->getReplacedEntryId());
+
+			foreach($metadataObjects as $sourceMetadataObject)
+			{
+					$metadataProfileId = $sourceMetadataObject->getMetadataProfileId();
+	
+					$targetMetadata = new Metadata();
+					$targetMetadata->setPartnerId($partnerId);
```

Comment:
Partner is and profile id might be different in case of cloning template partner default content.
Use kObjectCopyHandler::mapIds and kObjectCopyHandler::getMappedId to translate the profile.